cdp_scrapers/legistar_utils.py

```diff
@@ -507,6 +507,7 @@ def __init__(
         matter_rejected_pattern: str = r"rejected|dropped",
         minutes_item_decision_passed_pattern: str = r"pass",
         minutes_item_decision_failed_pattern: str = r"not|fail",
+        known_persons: Dict[str, Person] = None,
```

Comment:
Add a docstring for this new param and I believe it should be `Optional[Dict[str, Person]]`